README.md

```diff
@@ -62,9 +62,21 @@ MY_CLUSTER=...
 gcloud dataproc clusters create "$MY_CLUSTER"
 ```
 
-## Downloading the Connector
+## Downloading and Using the Connector
 
-The latest version connector of the connector is publicly available in [gs://spark-lib/bigquery/spark-bigquery-latest.jar](https://console.cloud.google.com/storage/browser/spark-lib/bigquery). A Scala 2.12 compiled version exist in gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar.
+The latest version connector of the connector is publicly available in 
+[gs://spark-lib/bigquery/spark-bigquery-latest.jar](https://console.cloud.google.com/storage/browser/spark-lib/bigquery).
+A Scala 2.12 compiled version exist in gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar.
```

Comment:
Usually artifacts prepend Scala version to artifact name, not version: `spark-bigquery_2.12-latest.jar.`